plexlibrary/recipe.py

```diff
@@ -75,31 +80,33 @@ def _get_trakt_lists(self):
         item_ids = []
 
         for url in self.recipe['source_list_urls']:
+            max_age = (self.recipe['new_playlist']['max_age'] if self.use_playlists
```

Comment:
Ugly exception on missing "mandatory" keys in the recipe. I'm guessing many will get stuck here:

```Traceback (most recent call last):
  File "/usr/lib64/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib64/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "plexlibrary/__main__.py", line 3, in <module>
    main()
  File "plexlibrary/plexlibrary.py", line 67, in main
    r.run(sort_only=args.sort_only, share_playlist_to_all=args.everyone)
  File "plexlibrary/recipe.py", line 652, in run
    missing_items, list_count = self._run(share_playlist_to_all=share_playlist_to_all)
  File "plexlibrary/recipe.py", line 587, in _run
    item_list, item_ids = self._get_trakt_lists()
  File "plexlibrary/recipe.py", line 83, in _get_trakt_lists
    max_age = (self.recipe['new_playlist']['max_age'] if self.use_playlists
  File "/home/adam/.local/lib/python3.6/site-packages/ruamel/yaml/comments.py", line 753, in __getitem__
    return ordereddict.__getitem__(self, key)
KeyError: 'max_age'
```

Reply:
Noticed a few other places like this too. Probably should have some sort of sanity check on the recipe depending on options used.

Reply:
```suggestion
            max_age = (self.recipe['new_playlist'].get('max_age') if self.use_playlists
```
max_age = None if max_age doesn't exist in the recipe

Reply:
Looks better, but we would still crash if there's no `new_playlist` in the yaml. I can't apply any suggestions or changes, github is annoying that way.